pkg/cmd/server/start/start_allinone.go

```diff
@@ -227,7 +228,7 @@ func (o AllInOneOptions) StartAllInOne() error {
 		if err != nil {
 			return err
 		}
-		fmt.Fprintf(o.MasterOptions.Output, "%s\n", host)
+		fmt.Fprintf(o.Output, "%s\n", host)
```

Comment:
Can revert as technically right now `o.MasterOptions.Output == o.Output` but as written previously this didn't really make sense, as this is output of `allinone`.